### ISSUE TITLE
[css-conditional-4] @supports elements queries

### ISSUE THREAD

**Issue by NagayamaToshiaki** [w3c/csswg-drafts#7023]
There have been not much changes or addition of elements recently, but inability of querying the support of HTML element is not handy.
so I would like to issue the new specs that would be written like this:

```
@supports element(foo) {
    foo {
        line-height: 1.5;
    }
}
```

**Comment by svgeesus** [comment]
I agree that the proposed syntax has an unfortunate clash with [the `element()` notation](https://www.w3.org/TR/css-images-4/#element-notation).

More concerning though is that we would need a crisp definition of what it means to _support an element_. Simply parsing it and not throwing it away? Something more, surely, but what exactly?
